tests/test_install_mender_sh.py

```diff
@@ -56,12 +64,7 @@ def _get_localhost_ip(self, setup_tester_ssh_connection):
     @pytest.mark.usefixtures("setup_test_container_f")
     @pytest.mark.parametrize("channel", ["", "stable", "experimental"])
```

Comment:
My suggestion in the task acceptance criteria is to only test on experimental until the release:
```suggestion
    @pytest.mark.parametrize("channel", ["experimental"])
```

Reply:
You're right. I was trying to be smart here by selectively testing each package, but it's tricky because `install-mender.sh` doesn't even execute correctly if `experimental` isn't set. I will just do what you said and skip, which also makes it easier to clean up later.